packages/framework/attributor/src/test/mixinAttributor.spec.ts

```diff
@@ -28,10 +28,6 @@ type Mutable<T> = {
     -readonly[P in keyof T]: T[P]
 };
 
```

Comment:
this is leftover misc. cleanup from some backed-out changes. Figured adding it here would be fine since it's effectively a test adjustment PR anyway.